Services/PackageService.cs

```diff
@@ -0,0 +1,36 @@
+using Repositories.Entities;
+using Repositories.Interfaces;
+
+namespace Services
+{
+    public class PackageService : IPackageService
+    {
+        private readonly IPackageRepository _packageRepository;
+        private readonly ICourierRepository _courierRepository;
+
+        public PackageService(IPackageRepository packageRepository, ICourierRepository courierRepository)
+        {
+            _packageRepository = packageRepository;
+            _courierRepository = courierRepository;
+        }
+
+        public async Task<List<Package>> GetUnassignedAsync()
+        {
+            var packages = await _packageRepository.GetAsync();
+            var couriers = await _courierRepository.GetAsync();
+            foreach (Courier courier in couriers)
```

Comment:
I think you can use var in foreach also? meaning var courier should be here I think